src/Compilers/CSharp/Portable/Symbols/SignatureOnlyMethodSymbol.cs

```diff
@@ -91,6 +91,8 @@ internal sealed class SignatureOnlyMethodSymbol : MethodSymbol
 
         public override string Name { get { return _name; } }
 
+        internal sealed override bool IsNullableAnalysisEnabled() => false;
```

Comment:
It feels like this should either be throwing unreachable or be settable in the constructor.